pkg/verifier/access.go

```diff
@@ -0,0 +1,26 @@
+package verifier
```

Comment:
IMHO, since `pkg/verifier` is primarily used for test verification, it should be located in `test/verifier`(or verifier.go within test dir) rather than `pkg/verifier`.

Reply:
we put it in pkg because the verifier is relied on otel sdk(In memory exporter) and the sdk in the commercial version is in pkg directory.